src/mpcCoreKit.ts

```diff
@@ -43,12 +43,14 @@ import {
   InitParams,
   JWTLoginParams,
   MPCKeyDetails,
+  NewTSSLibType,
```

Comment:
should we rename to v4TSSLibType to be more precise
as there will be always have new TssLibType

Reply:
alright can do this

Reply:
done